torch/onnx/__init__.py

```diff
@@ -138,6 +137,36 @@
 producer_name = "pytorch"
 producer_version = _C_onnx.PRODUCER_VERSION
 
+_CUSTOM_ONNX_DECOMPS = {}
+
+
+def register_decomp(
+    target: Callable, onnx_decomp: Callable, is_complex: bool = False
+) -> None:
+    """Register a function to decompose a PyTorch node to ONNX.
+
+    Args:
+        target: The PyTorch node callable target. E.g. ``torch.ops.aten.add``.
+        onnx_decomp: The ONNX decomposition function to register.
+        is_complex: Whether the function is a function that handles complex valued inputs.
```

Comment:
Is there any better explanation of how complex functions are handled? Without a more detailed explanation, it seems hard to use it correctly (or, it might end up being used incorrectly). Or, a pointer to the current code handling it would be a useful starting point for me to understand it better.